### ISSUE TITLE
Human-Robot Gym: Benchmarking Reinforcement Learning in Human-Robot Collaboration. (arXiv:2310.06208v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#28057]
Human-Robot Gym: Benchmarking Reinforcement Learning in Human-Robot Collaboration. (arXiv:2310.06208v1 [cs.RO])<br>
https://ift.tt/iNMK1jk<br>
<p>Deep reinforcement learning (RL) has shown promising results in robot motion
planning with first attempts in human-robot collaboration (HRC). However, a
fair comparison of RL approaches in HRC under the constraint of guaranteed
safety is yet to be made. We, therefore, present human-robot gym, a benchmark
for safe RL in HRC. Our benchmark provides eight challenging, realistic HRC
tasks in a modular simulation framework. Most importantly, human-robot gym
includes a safety shield that provably guarantees human safety. We are,
thereby, the first to provide a benchmark to train RL agents that adhere to the
safety specifications of real-world HRC. This bridges a critical gap between
theoretic RL research and its real-world deployment. Our evaluation of six
environments led to three key results: (a) the diverse nature of the tasks
offered by human-robot gym creates a challenging benchmark for state-of-the-art
RL methods, (b) incorporating expert knowledge in the RL training in the form
of an action-based reward can outperform the expert, and (c) our agents
negligibly overfit to training data.
</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
